commons/src/test/java/com/powsybl/commons/util/StringToIntMapperTest.java

```diff
@@ -53,6 +57,60 @@ void test() throws IOException {
         }
     }
 
+    @Test
+    void testDump() throws IOException {
+        try (FileSystem fileSystem = Jimfs.newFileSystem(Configuration.unix())) {
+            Path testDir = fileSystem.getPath("/tmp");
+            Files.createDirectories(testDir);
+            Path outputFile = testDir.resolve("output.txt");
+
+            StringToIntMapper<TestSubset> mapper = new StringToIntMapper<>(TestSubset.class);
+            String value = "value1";
+            mapper.newInt(TestSubset.TYPE, value);
+            mapper.dump(outputFile);
+
+            String expectedStr = "TYPE;value1;1\n";
```

Comment:
This test may fail on some OS because of the `\n`. Use `System.lineSeparator()` instead.